testing/renderer-with-debug-ui.tsx

```diff
@@ -28,22 +31,42 @@ export const RendererWithDebugUI = ({
     registerAllWidgetsForTesting();
     const ref = React.useRef<Renderer | null | undefined>(null);
     const [state, setState] = React.useState<any>(null);
+    const [isMobile, setIsMobile] = React.useState(false);
 
     return (
         <SideBySide
-            leftTitle="Widget"
+            leftTitle={
+                <View
+                    style={{
+                        flexDirection: "row",
+                        alignItems: "center",
+                        width: "100%",
+                    }}
+                >
+                    Widget
+                    <View style={{marginLeft: "auto"}}>
```

Comment:
This was the only way I could figure out pushing the switch to the right side.  `alignSelf: 'end'` didn't work. 😞

Reply:
`marginLeft` is a fine solution. Another thing I often do in flexbox layouts is add a blank element between the left-aligned and right-aligned things, and set it to `flexGrow: 1` so it expands to fill the space.